src/main/java/com/revature/controllers/ContentController.java

```diff
@@ -92,7 +92,11 @@
 	public ResponseEntity<Set<Link>> getLinksByContentId(@PathVariable int id) {
 		return ResponseEntity.ok(contentService.getLinksByContentId(id));
 	}
-
+	// return list of links attached to a given curriculum
+	@GetMapping("/curricula/{id}/links")
```

Comment:
This would come out to be /content/curricula/{id}/links. We shouldn't need /content/

Reply:
perhaps move this method to the curriculum controller?

Reply:
the reason i put it on content is that the front end used this method to extract content from links

Reply:
if i moved it to curriculum controller  the mapping will be : 
/content/{id}

Reply:
Why would content be in it at all? The class is annotated with @RequestMapping("/curricula") and you could have a method which has a @GetMapping with "/{id}/links"

Reply:
you right it useless to include content, i'll move it to curriculum controller